bindings/bindings-linux-aarch64-R4.2.rs

```diff
@@ -284,11 +284,14 @@ pub struct _IO_FILE {
     pub _mode: ::std::os::raw::c_int,
     pub _unused2: [::std::os::raw::c_char; 20usize],
 }
-#[doc = ", MAYBE"]
```

Comment:
This is some weird comment likely inferred from the header file. Can we drop/override it specifically for this one enum?

Reply:
Unfortunately, I cannot. these are not modifiable. Eventually, I'd figure out a good system to snap the comments, but it is just difficult with the chaos that is R-API's comments..